Afoster va 13 jan rl (#938)
department-of-veterans-affairs/developer-portal: src/content/apiDocs/health/fhirAPIReleaseNotes.mdx

### January 13, 2022
Added FHIR R4 `Encounter` Resource data which includes the ability to search by `patient`, `date` and by `_lastUpdated` date.

`vaccineCode` field in the R4 `Immunization` resource has been enhanced to include Vaccine Short Description as well as Vaccine Group Code and Name based on the [CDC CVX to Vaccine Groups code set](https://www2a.cdc.gov/vaccines/iis/iisstandards/vaccines.asp?rpt=vg).

> Added support for `_lastUpdated` search for R4 `Observation` resource.

Reviewer: Just to remove the back to back fors in the sentence, I inserted "in the".
```suggestion
Added support for `_lastUpdated` search in the R4 `Observation` resource.
```

---
### January 10, 2022

> We changed the `referenceRange` that's returned with an `Observation` resource.

Reviewer: When I first read this, I wasn't sure if the bullet points were referring to 'referenceRange' or 'Observation resource.' I tried to clarify this a bit with the revised sentence. 

```suggestion
We changed the `referenceRange` data field that returns an `Observation` resource.
```

> We changed the `referenceRange` that is returned with the `Observation` resource.

Reviewer: ```suggestion
We changed the `referenceRange` data field that is returned with the `Observation` resource.
```

 - It no longer returns the field `high` or `low` if there is not an associated value.

>  - It now returns the `system`, `code`, and `unit` only if all data are present.   

Reviewer: I think it might help the clarity of the sentence to insert the word fields after listing system, code, and unit.
```suggestion
 - It now returns the `system`, `code`, and `unit` fields only if all data are present.   
```

---
### December 13, 2021

> Added support for `_lastUpdated` search for DSTU2 and R4 `Procedure` resource.

Reviewer: Added "in both the" before DTSU2 and R4.

```suggestion
Added support for `_lastUpdated` search in both the DSTU2 and R4 `Procedure` resource.
```

Integrated a new data source that contains COVID Vaccination data for VA Employees who received their vaccinations at a VA Facility.  This data is now available in the `Immunization` resource.  

---
### November 16, 2021
Added `qualification.code.text` and `address.period` data fields to the `Practitioner` resource.

> Added the `code.text` data field to the ` PractitionerRole` resource.  

Reviewer: I removed the space between the ' and the P, which was starting the word PractitionerRole. 

```suggestion
Added the `code.text` data field to the `PractitionerRole` resource.  
```
